velox/type/Timestamp.h

```diff
@@ -33,9 +33,14 @@ namespace facebook::velox {
 struct Timestamp {
  public:
   enum class Precision : int { kMilliseconds = 3, kNanoseconds = 9 };
+  // nanos in Timestamp should not greater than 1 second.
```

Comment:
Comments should be full sentences, e.g. start with a capital letter. Please, review coding style guidelines and update accordingly.